src/shared/src/types/work-package-types.ts

```diff
@@ -17,3 +19,29 @@ export enum WorkPackageStage {
   Install = 'INSTALL',
   Testing = 'TESTING'
 }
+
+export interface Blocked_By_Info {
+  blockedByInfoId: string;
+  stage?: WorkPackageStage;
+  name: string;
+  workPackageTemplate: WorkPackageTemplate;
```

Comment:
remove template and templateId from this type

Reply:
@RChandler234 why don't we need these types here?

Reply:
we don't ever need to access the template from a team type